packages/bun/package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "@sentry/bun",
+  "version": "7.69.0",
+  "description": "Official Sentry SDK for bun",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/bun",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8"
+  },
+  "main": "build/cjs/index.js",
```

h: This currently points to non-existing CJS:

I guess it should be safe to just point to esm here 🤔 

```suggestion
  "main": "build/esm/index.js",
```